lib/helpers.js

```diff
@@ -18,3 +20,13 @@ export function getEditorsMap(editorUI) {
   })
   return {editors, diff, activePaths}
 }
+
+export function getMessageRange(message) {
+  return message.range ? (
+    message.range.constructor.name === 'Array' ? Range.fromObject(message.range) : message.range
```

Comment:
`Range.fromObject` can accept a `Range`, in which case it just returns it. You might as well just always call that.

(Although this _does_ avoid a function call...)